environment.conf.json

```diff
@@ -101,4 +101,4 @@
   "SEARCH_ENDPOINT": "https://acc.api.data.amsterdam.nl/signals/v1/private/search",
   "TERMS_ENDPOINT": "https://acc.api.data.amsterdam.nl/signals/v1/private/terms/categories/",
   "USERS_ENDPOINT": "https://acc.api.data.amsterdam.nl/signals/v1/private/users/"
-}
+}
```

Comment:
Here is the newline missing, is this intended?

Reply:
This was VSCode's intention, not sure why. Some linting rules have changed?

Reply:
Fixed it.